docs/Plugins-Guide.md

```diff
@@ -271,6 +271,28 @@ module.exports = fp(dbPlugin)
 ```
 You can also tell to `fastify-plugin` to check the installed version of Fastify, in case of you need a specific api.
 
+As we mentioned earlier, Fastify starts loading its plugins __after__ `.listen()`, `.inject()` or `.ready()` are called and as such, __after__ they have been declared. This means that, even though the plugin may inject variables to the global scope via `decorate`, the decorated variables will not be accessible before calling `.listen()`, `.inject()` or `.ready()`.
```

Comment:
The global scope is  `global`.

> Event though the plugin may call `decorate` on the current instance of fastify,  the decorated properties...

(decorate must be a link that points to the decorator guideline).

Reply:
Reworded once again as per your first comment and added link to the guideline.